types/jasmine-ajax/index.d.ts

```diff
@@ -5,14 +5,10 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-interface JasmineAjaxResponse {
-    status?: number;
-    statusText?: string;
-    responseText?: string;
-    response?: string;
-    contentType?: string;
-    responseHeaders?: { [key: string]: string };
-}
+/**
+ * @deprecated Use JasmineAjaxRequestStubReturnOptions instead
+ */
+interface JasmineAjaxResponse extends JasmineAjaxRequestStubReturnOptions {}
```

Comment:
I'd like to remove this type in favor of `JasmineAjaxRequestStubReturnOptions`, but it will be a breaking change, so I've left it and marked it as deprecated.